client/src/components/mobile/WhatsUp.js

```diff
@@ -158,6 +203,9 @@ export default function WhatsUp({ chat }) {
                 </Message>
               </MessageWrapper>
             ))}
+            <WritingContainer onSubmit={handleSubmit}>
+              <WritingArea placeholder="Schreib etwas" /> <Send />
```

Comment:
I would recommend making the component controlled, by giving it a value with a state and an onchange function, that sets the state of the component.